src/doom/r_plane.c

```diff
@@ -484,7 +485,16 @@ void R_DrawPlanes (void)
 
 		if ((unsigned) dc_yl <= dc_yh) // [crispy] 32-bit integer math
 		{
-		    angle = ((an + xtoviewangle[x])^flip)>>ANGLETOSKYSHIFT;
+			// [crispy] Draw skies horizontally linear. Taken from GZDoom.
+			if (crispy->linearsky) 
+			{
+				angle_t xangle = (angle_t)((0.5 - x / (double)viewwidth) * tan(-90) * ((double)SCREENWIDTH/NONWIDEWIDTH) * 0.5 * ANG90);
```

Comment:
`tan(-90)` ?

Reply:
Haha, when I was adapting the GZDoom code I first thought I had to adjust the angle based on a tangent of the fov ratio, and forgot to take it out... what can I say, it's been a while since I've done any trigonometry! :p 
(I'm a bit surprised the compiler didn't even throw a warning...)

Reply:
Sorry for the sparse reply. First, thanks for taking the time to implement this for Crispy!

However, I am almost sure we can get away without floating point math in this calculation. 

Then, `tan()` with a constant argument returns a constant value. Moreover, `tan()` takes its arguments in radians, not degree, so `-90` is almost certainly not the value you are looking for.

Reply:
>tan() takes its arguments in radians, not degree

Ah, I was just assuming that tan(-90) would be undefined, forgetting it wasn't in degrees. In any case I did gather that it was an unneeded constant and already removed it. 

>I am almost sure we can get away without floating point math in this calculation.

OK, will fix this.